get_goes_x.py

```diff
@@ -31,33 +33,32 @@
     print(('Warning: failed to open URL {}: {}'.format(url, last_err)))
     sys.exit(0)
 
-
-colnames = 'year month dom  hhmm  mjd secs short long ratio'.split()
-data_colnames = colnames[-3:]
-
 try:
-    dat = ascii.read(urldat, guess=False, Reader=ascii.NoHeader,
-                     data_start=3, names=colnames)
+    dat = Table(json.loads(urldat))
 except Exception as err:
     print(('Warning: malformed GOES_X data so table read failed: {}'
           .format(err)))
     sys.exit(0)
 
-# Strip up to two rows at the end if any values are bad (i.e. negative)
-for _ in range(2):
-    if any(dat[name][-1] < 0 for name in data_colnames):
-        dat = dat[:-1]
-
-mjd = dat['mjd'] + dat['secs'] / 86400.
+# Select only the GOES 16 data (which is all there is)
```

Comment:
Satellite number should be a command line option so we can change in the future with only an update to task schedule.

Reply:
That seemed like overconfigurating to me based on how not-robust the rest of this is.

Reply:
But I did it anyway.